### ISSUE TITLE
Publish with Strong Name

### ISSUE THREAD

**Issue by AnakinRaW** [shravan2x/Gameloop.Vdf#31]
Hello there,
do you have any plans publishing the nuget package with a strong name? Currently I can't use this package because it's not signed.

**Comment by shravan2x** [comment]
Hi, I tried to do this for the last release but the feature wasn't very stable yet. Let me try again today.

**Comment by shravan2x** [comment]
From what I understand, there are both pros and cons to publishing with a strong name. I want to be careful with this change so existing references don't break (or minimize the damage). I'm looking into how other common packages handle this.

**Comment by AnakinRaW** [comment]
AFAIK there should be no problems just signin it. A signed library can be used by non-signed libraries. Just a singed application must always use signed libs. 
Also afaik most common approach is just to publish the .SNK file with your repository. Just make sure that snk file is not share with other libraries. According to MS a strong name is *not* intended for application secuirity anyway, so that should be OK.
